package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-number-editor",
-  "version": "4.0.2",
+  "version": "4.0.3",
```

Comment:
Could you revert this line please? :)
A new version will be set when releasing the package.